import std/core;

@[extern_name: "get_errno"]
func errno() -> &&Error = extern;
